### DISCUSSION TITLE
SVG version support

### DISCUSSION THREAD

**Discussion by P4bloo** [mono/SkiaSharp#2655] [Q&A]
Hi folks,

I attempted to read an SVG object from a file, scale it, and then save it as an image using canvas.DrawPicture. Within my SVG definition, I utilized vector-effect="non-scaling-stroke" to maintain an unchanged stroke during scaling.

However, the scaled object in the output file appears to have a stroke that is still influenced by scaling (it appears that vector-effect="non-scaling-stroke" was not considered).

What SVG version is supported by SkiaSharp 1.1/2.0?

**Comment by mattleibow** [comment]
How are you loading the SVG? SkiaSharp does not have native SVG support so the loading library may be affecting this.
